src/render.rs

```diff
@@ -229,6 +229,16 @@ impl Render for Filter {
                     }
                 }
             }
+            FilterType::AddSlashes => match left {
+                Some(content) => Some(Content::String(Cow::Owned(
+                    content
+                        .render(context)?
+                        .replace("\\", "\\\\")
```

Comment:
Like Python, [Rust also supports raw strings](https://doc.rust-lang.org/rust-by-example/std/str.html#literals-and-escapes):

```suggestion
                        .replace(r"\", r"\\")
```

Reply:
Fixed